Elikdev/Gatepass
# users get all registered applications

addition of:  
 - Users to view all apps on their organuzation
   - - users can also filter by active or disabled apps

**What does this PR do?**
Fixes #11 

**description of Task to be completed?**
- Users to update their applications on gatepass

**How should this be manually tested?**
- Check out to this branch `git checkout feat-getapps`
- run `npm run devStart`
- Test these endpoint on gatepass:
     - `/api/v1/apps` or `/api/v1/apps?filter=active` or` /api/v1/apps?filter=disabled`
     -  or `/api/v1/apps?page=1&limit=10&filter=active` or `like this /api/v1/apps?page=1&limit=1`

**Any background context you want to provide?**
- None


**What is the link to the issue on Github?**

[Issue #11](https://github.com/Elikdev/Gatepass/issues/11)

**Questions:**
- None
If something is unclear or you want some questions to be addressed by your peers, mention them here
